### PR TITLE
feat: add Static Legend

### LINKED ISSUE #806: Static Legend: Integrate Giraffe changes into UI
### Background
We need to integrate the giraffe Legend work into the UI

### Acceptance Criteria
- [ ] We're on the latest version of giraffe that includes Legends
- [ ] as a user, when i configure a graph or band graph in the UI, then I have the option to always display a legend on the bottom of the cell.
- [ ] as a user, when i enable the legend on a cell, no changes to hover tooltip display when hovering over different parts of the graph
- [ ] as a user, I can choose to configure whether recent time values will be displayed in the legend
- [ ] as a user, I can choose the height of the static legend in a configuration option

### LINKED ISSUE #1134: Static Legend: e2e tests
Create end to end tests for static legend.

- [x] Test that an option exists and works for turning on or off the static legend exists for Line Graph and Band Plot.
- [x] Test that an option does **_not_** exist for turning on or off the static legend for any other graph types not listed above.
- [x] Test that an option exists and works for turning on or off showing the latest value for each series in the static legend.
- [x] Test that an option exists and works to change the height of the static legend.

### PR DESCRIPTION
**Pull request by TCL735**
Closes #806 
Closes #1134 

Screen Shot 2021-05-18 at 4 31 42 PM

Screen Shot 2021-05-18 at 4 31 31 PM

Screen Shot 2021-05-18 at 4 56 35 PM

Screen Shot 2021-05-18 at 7 12 00 PM

### CODE REVIEW

**Comment by ui-ci-bot-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 55 | :red_circle:
accessibility | 96 | :green_circle:
seo | 90 | :green_circle:
pwa | null | :hankey:
best-practices | 93 | :green_circle:


[Compare your scores with master branch](https://influxdb.aws.influxdata.io/orgs/844910ece80be8bc/dashboards/073c4b95f1089000?lower=now%28%29%20-%202d&vars%5Blighthouse_score_branch%5D=1448)

**Comment by TCL735** [comment]
Force push to fix merge conflicts

**Comment by TCL735** [comment]
Ready for review

**Review by zoesteinkamp — APPROVED** [review]
This all looks good, is this work behind a feature flag? Seems like a good chunk of new features

**Comment by TCL735** [comment]
Yes it is feature flagged under `staticLegend`

**Review by jrenee42 — CHANGES_REQUESTED** [review]
minor changes so far...haven't actually finished my review yet.

just posting so you can see what I have done so far

File: cypress/e2e/shared/explorer.test.ts [hunk 1]
```diff
@@ -1061,6 +1061,94 @@ describe('DataExplorer', () => {
         })
       })
     })
+
```
[the remark this answers is not in the document]
Comment (TCL735) [reply cypress/e2e/shared/explorer.test.ts:1064]: Sure. But I'll make it read like a grammatically correct sentence. it: turns static legend flag off, so static legend option should not exist'

**Pull request merged**